fastlane/Fastfile

```diff
@@ -5,7 +5,7 @@ platform :ios do
   before_all do 
     # Perform a fetch before inferring the next version 
     # to reduce race conditions with simultaneous pipelines attempting to create the same tag
-    sh('git', 'fetch', '--tags')
```

Comment:
Adding to forcefully fetch all tags because of the [failing deploy unstable task](https://app.circleci.com/pipelines/github/aws-amplify/amplify-swift/7586/workflows/c4a15a2e-a091-4163-8cf0-69e5c5bdcc9d/jobs/55195) on CircleCi